packages/kbn-optimizer/limits.yml

```diff
@@ -107,7 +107,7 @@ pageLoadAssetSize:
   dataVisualizer: 27530
   banners: 17946
   mapsEms: 26072
-  timelines: 330000
+  timelines: 327300
```

Comment:
@jbudz FIY, I managed to reduce the bundle size (with the help of @kqualters-elastic) under the previous limit value